systemd/crc-needs-tap.sh

```
@@ -0,0 +1,26 @@
+    echo "Running with Apple virtualization. Need tap0."
+    exit $NEED_TAP
+fi
+
+if /usr/local/bin/crc-self-sufficient-env.sh; then
+    echo "Running with a self-sufficient bundle. Don't keep tap0"
+    exit $DONT_NEED_TAP
+fi
+
+echo "No particular environment detected. Don't keep tap0"
+
+exit DONT_NEED_TAP
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)